include/jemalloc/internal/cache_bin.h

```diff
@@ -33,6 +33,12 @@ static const uintptr_t cache_bin_preceding_junk =
 /* Note: a7 vs. 7a above -- this tells you which pointer leaked. */
 static const uintptr_t cache_bin_trailing_junk =
     (uintptr_t)0xa7a7a7a7a7a7a7a7ULL;
+/*
+ * A pointer used to initialize a fake stack_head for disabled small bins
+ * so that the enabled/disabled assessment does not entirely rely on
+ * ncached_max.
+ */
+extern const void * const disabled_bin;
```

Comment:
Let's abstract out with a wrapper, and use the address of disabled_bin, since it's defined in cache_bin.c which is a unique copy:

```
const void * cache_bin_diable_bin_stack() {
  return &disabled_bin;
}
```

Reply:
Sure, if that's the case, disabled_bin itself does not have to be a pointer, I make it a `uintptr_t` just like the `cache_bin_preceding_junk`.